### PR TITLE
Vue example

### PR DESCRIPTION
**Pull request by dxArtemiusz**

### CODE REVIEW

File: Vue/src/TooltipTemplate.vue [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    isDeleteButtonExist: {
+      type: Boolean
+    }
+  },
+  data() {
+    return {
+      employees,
+    };
+  },
+  methods : {
```
Comment (sergepilipchuk) [inline Vue/src/TooltipTemplate.vue]: Please, fix spaces

File: Vue/src/App.vue [hunk 2]
```diff
@@ -1,9 +1,130 @@
...
+      dataSource: data,
+      schedulerRefKey
+    };
+  },
+  computed: {
+    scheduler: function() {
+      return (this.$refs[schedulerRefKey] as any).instance;
+    }
+  },
+  methods:{
```
Comment (sergepilipchuk) [inline Vue/src/App.vue]: Please, fix spaces

Comment (dxArtemiusz) [reply Vue/src/App.vue]: Fixed

File: Vue/src/TooltipTemplate.vue (hunk 1 above)
Comment (dxArtemiusz) [reply Vue/src/TooltipTemplate.vue]: Fixed

**Review by flagmanAndrew — APPROVED** [review]

**Pull request merged**